pkg/tbtc/dkg_submit_test.go

```diff
@@ -283,6 +379,13 @@ func TestSubmitResult_MemberDoesNotSubmitsResult(t *testing.T) {
 	ctx, cancelCtx := context.WithCancel(context.Background())
 	defer cancelCtx()
 
+	// Set up a global listener that will cancel the common context upon result
+	// submission. That mimics the real-world scenario.
+	localChain.OnDKGResultSubmitted(
+		func(event *DKGResultSubmittedEvent) {
+			cancelCtx()
+		})
+
 	secondMemberSubmissionChannel := make(chan error)
 
 	// Attempt to submit result for the second member on a separate goroutine.
```

Comment:
Not something we need to worry about now but... why does it have to be a separate goroutine? Can't we do it synchronously? We `time.Sleep` one line later to have this code executed first.

Reply:
We need to make member 2 waiting for its submission turn and member 1 must actually submit during member 2 wait time. We can't do that without the goroutine.